mods/voidmulti.sc2mod/base.sc2data/gamedata/sounddata.xml

```diff
@@ -270,6 +270,12 @@
         <VolumeRolloffPoints Distance="10.000000"/>
         <VolumeRolloffPoints Distance="20.000000" Volume="-96.000000"/>
     </CSound>
+    <CSound id="Interceptor_AttackImpact">
+        <Volume value="-9.000000,-8.000000"/>
+    </CSound>
+    <CSound id="Interceptor_AttackLaunch">
+        <Volume value="-8.000000,-5.000000"/>
```

Comment:
**undocumented**: Interceptor attack firing sounds are slightly louder (Volume min,max increased from -10,-6 to -8,-5)